src/components/new-request/business-lookup.vue

```diff
@@ -77,17 +80,43 @@ export default class BusinessLookup extends Vue {
   /** State of this component. */
   state = States.INITIAL
 
+  /** Business Lookup labels and warnings. */
+  lookupLabel = ''
+  lookupNoActiveText = ''
+
+  /** Set business lookup text on mount. */
+  mounted (): void {
+    this.setLookupText()
```

Comment:
You may be able to delete this and add "immediate: true" to the watcher instead.

Reply:
I like this a lot Sev. Never knew that. It works!

Reply:
👍 

FYI, the other useful watcher option is "deep" (to watch changed object properties, but there are caveats).

Reply:
Looks good.